assets/components/contributionsCheckout/contributionsCheckout.scss

```diff
@@ -9,6 +9,13 @@
     margin-top: $gu-v-spacing * 3;
   }
 
+  .component-your-details-text {
```

Comment:
This is a [BEM element](http://getbem.com/naming/), so `component-your-details__text`.